packages/xo-server/src/api/host.js

```diff
@@ -2,6 +2,46 @@ import { format } from 'json-rpc-peer'
 
 // ===================================================================
 
+export async function getSchedulerGranularity({ host }) {
+  let schedGran
+  try {
+    schedGran = await this.getXapi(host).getField('host', host.id, 'sched_gran')
+  } catch (e) {
+    schedGran = e.code
+  } finally {
+    return schedGran
+  }
+}
+
+getSchedulerGranularity.description = 'get the scheduler granularity of an host'
+
+getSchedulerGranularity.params = {
+  host: { type: 'object' },
+}
+
+getSchedulerGranularity.resolve = {
+  host: ['id', 'host', 'view'],
+}
+
+// ===================================================================
+
+export function setSchedulerGranularity({ host, value }) {
+  this.getXapi(host).setField('host', host.id, 'sched_grand', value)
+}
+
+setSchedulerGranularity.description = 'set scheduler granularity of an host'
+
+setSchedulerGranularity.params = {
+  host: { type: 'object' },
+  value: { type: 'string' },
```

Comment:
Better use named API props it will help to understand what should we pass as args.